conf/machine/include/imx-base.inc

```diff
@@ -158,7 +158,6 @@ MACHINE_SOCARCH_FILTER_append_imx = " \
 "
 MACHINE_SOCARCH_FILTER_append_imxvpu = " \
     imx-codec \
-    imx-parser \
```

Comment:
@thochstein how so? could you explain why the change?

Reply:
There's nothing machine or SOC-specific in the recipe?